program/src/instructions/close_mint_proof_v2.rs

```diff
@@ -24,6 +24,7 @@ pub struct CloseMintProofV2<'info> {
     pub signer: Signer<'info>,
 
     /// The mint proof account to close.
+    /// TODO: missing seeds?
```

Comment:
No need for seeds check here; any mint proof account can be closed after the slot delay or by the original payer before that.